.pre-commit-config.yaml

```
@@ -3,12 +3,12 @@ repos:
     rev: 24.3.0
     hooks:
       - id: black
-        language_version: python3.11
+        language_version: python3.10
```

We upgraded the python version used by precommit in #1482. I wouldn't downgrade it to 3.10 here. Is there any reason for this change? If not, feel free to restore these lines to 3.11:
```suggestion
        language_version: python3.11
```